feature/notification/src/main/java/io/github/droidkaigi/confsched2020/notification/worker/ManageTopicSubscriptionWorker.kt

```diff
@@ -0,0 +1,105 @@
+package io.github.droidkaigi.confsched2020.notification.worker
+
+import android.content.Context
+import androidx.work.Constraints
+import androidx.work.Data
+import androidx.work.ExistingWorkPolicy
+import androidx.work.ListenableWorker
+import androidx.work.NetworkType
+import androidx.work.OneTimeWorkRequestBuilder
+import androidx.work.WorkManager
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import com.google.firebase.iid.FirebaseInstanceId
+import com.google.firebase.messaging.FirebaseMessaging
+import io.github.droidkaigi.confsched2020.notification.Topic
+import io.github.droidkaigi.confsched2020.timber.error
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.tasks.await
+import timber.log.Timber
+import timber.log.debug
+
+class ManageTopicSubscriptionWorker(
```

Comment:
Thanks!
I think this is not a related code.
Class for receiving announcements from DroidKaigi.

Reply:
I think most of the code to receive from firebase.
Can that part be removed once? 🙇

Reply:
Thank you for your advice！
I haven't made a pull request so much ... I deleted something I thought was unnecessary and pushed it. Is the change reflected?

Reply:
I noticed by email, do I need to delete the same for the woker and service classes?

Reply:
Yes. The task of receiving the announcement has been assigned to another person. 🙇 
https://github.com/DroidKaigi/conference-app-2020/issues/145

Reply:
The fix is ​​complete, is this OK?